cumulusci/cli/cci.py

```diff
@@ -851,6 +856,15 @@ def org_import(config, username_or_alias, org_name):
     )
 
 
+def calculate_org_days(info):
+    """Returns the difference in days between created_date (ISO 8601),
+    and expiration_date (%Y-%m-%d)"""
+    created_date_str = info["created_date"].split("T")[0]
```

Comment:
Not sure if there's a better way to easily convert the ISO datetieme to a simple date.

Reply:
datetime_format = "%Y-%m-%dT%H:%M:%S.%f+0000"
will work

Reply:
well, I'm already using strptime() to process that date portion, which is all I want. I guess I was looking to see if there's a one-liner that converts the ISO formatted string to a date.

Reply:
Gotcha, not sure about that one.